tests/js/spec/views/organizationStats/index.spec.jsx

```diff
@@ -303,10 +310,9 @@ describe('OrganizationStats', function () {
       }),
     });
 
-    const paginate = wrapper.find('Pagination');
-    paginate.props().onCursor('0:100:0');
```

Comment:
This wasn't _really_ testing anything before, since it was just calling out to the default prop variant. I kept it around just in case but turned it into an actual component test by simulating the click.